ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    project = "[your-project-id]"
</pre><p>We typically use PROJECT_ID as the var name in the other notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>